pipelines/_ingredients/cpLinearizerLSST.yaml

```diff
@@ -0,0 +1,63 @@
+description: cp_pipe linearizer calibration construction.
+tasks:
+  cpLinearizerIsr:
+    class: lsst.ip.isr.IsrTaskLSST
+    config:
+      connections.ccdExposure: "raw"
+      connections.outputExposure: "cpLinearizerIsrExp"
+      expectWcs: false
+      doDefect: true
+      doBias: false
+      doDark: false
+      doFlat: false
+      doDiffNonLinearCorrection: false
+      doBootstrap: true
+      doDeferredCharge: false
+      doBrighterFatter: false
+      doLinearize: false
+      doCorrectGains: false
+      doApplyGains: false
```

Comment:
To be clearer, the linearizer pipeline comes first.  There are no gains at this point.  One thing to note is that the natural fitting unit for the linearizer is ADU because the gain gets washed out with the slope of the mean vs photodiode or exptime.  The EXPAPPROXIMATION is just there as a filler in this pipeline.  We use the `linearizerPtc` merely as a summary object that holds all the inputs means and metadata so that the linearity fitting code can be run.  But the actual fit that comes out of this is immaterial.